packages/react/src/Footer/index.ts

```diff
@@ -1,4 +1,3 @@
 export { Footer } from './Footer'
 export type { FooterProps } from './Footer'
-export type { FooterBottomProps } from './FooterBottom'
-export type { FooterTopProps } from './FooterTop'
+export type { FooterSpotlightProps } from './FooterSpotlight'
```

Comment:
We probably also want to export `FooterMenuProps` and `FooterMenuLinkProps` here

Reply:
Added